vms/platformvm/block/builder/builder.go

```diff
@@ -270,59 +283,74 @@ func (b *builder) PackAllBlockTxs() ([]*txs.Tx, error) {
 		return nil, fmt.Errorf("could not calculate next staker change time: %w", err)
 	}
 
+	recommendedPChainHeight, err := b.txExecutorBackend.Ctx.ValidatorState.GetMinimumHeight(context.TODO())
```

Comment:
that could be a very dumb question but why don't we use preferredHeight instead?

Reply:
`GetMinimumHeight` is estimating the height that will be used by the proposervm